hydra/plugins/sweeper.py

```diff
@@ -63,3 +64,11 @@ def validate_batch_is_legal(self, batch: Sequence[Sequence[str]]) -> None:
             config_loader.load_sweep_config(
                 master_config=self.config, sweep_overrides=list(overrides)
             )
+
+    def _parse_sweeper_params_config(self) -> List[str]:
+        params_conf = []
+        assert self.params is not None
+        for k, v in self.params.items():
+            param = f"{k}={v}".replace("'", "")
```

Comment:
Why remove the single quotes?
I think single quotes could be useful sometimes, e.g. to distinguish between the dict `{1: 2}` and the string `'{1: 2}'` (see our docs on [quoting values](https://hydra.cc/docs/next/advanced/override_grammar/basic/#quoted-values)).